### PR TITLE
add ndslice to the changelog

### PR DESCRIPTION
**Pull request by 9il**

### CODE REVIEW

File: changelog.dd [hunk 1]
```diff
@@ -7,6 +7,8 @@ $(COMMENT Pending changelog for 2.069. This will get copied to dlang.org and
 $(BUGSTITLE Library Changes,
 
 $(LI $(RELATIVE_LINK2 json-encode-control-characters, `std.json` encodes string control characters.))
+$(LI The package $(STDMODREF experimental_ndslice, std.experimental.ndslice) was added.
+It is also availble at the $(LINK2 https://github.com/DlangScience/mir, Mir library).)
```
Comment (schuetzm) [inline changelog.dd]: typo: available
besides: maybe use "as part of" instead of "at"? sounds better to me

Comment (9il) [inline changelog.dd]: Thanks!

File: changelog.dd [hunk 2]
```diff
@@ -7,6 +7,8 @@ $(COMMENT Pending changelog for 2.069. This will get copied to dlang.org and
 $(BUGSTITLE Library Changes,
 
 $(LI $(RELATIVE_LINK2 json-encode-control-characters, `std.json` encodes string control characters.))
+$(LI The package $(STDMODREF experimental_ndslice, std.experimental.ndslice) was added.
+It is also available as part of $(LINK2 https://github.com/DlangScience/mir, Mir library).)
```
Comment (Hackerpilot) [inline changelog.dd]: Should be `part of the`

Comment (9il) [inline changelog.dd]: Fixed, thanks!

**Comment by Hackerpilot** [comment]
Auto-merge toggled on

**Pull request merged**